pages/projects/index.js

```diff
@@ -15,25 +16,39 @@ import {
 import { getNextStaticProps } from '@faustwp/core';
 import { pageTitle } from 'utilities';
 import { BlogInfoFragment } from 'fragments/GeneralSettings';
+import appConfig from 'app.config';
 
 export default function Page() {
```

Comment:
Is there a reason why we are using a Next.js file based page instead of the template hierarchy solution here?

Reply:
@blakewilson Not that I'm aware of, more so a remnant of legacy faust